library/modules/Units.cpp

```diff
@@ -528,7 +528,7 @@ bool Units::isFortControlled(df::unit *unit)
 {   // Reverse-engineered from ambushing unit code
     CHECK_NULL_POINTER(unit);
 
-    if (!gamemode || *gamemode != game_mode::DWARF)
+    if (*gamemode != game_mode::DWARF)
```

Comment:
They can be null, if they're missing from symbols.xml. These days, we tend to have all globals available to us, so that is less of a concern, and there is almost certainly lots of code that assumes globals aren't null. So while I wouldn't remove null checks from existing code, I think it's fine to omit them in new code.